ocaml/xcp-rrdd/bin/rrdp-cpu/rrdp_cpu.ml

```diff
@@ -106,6 +107,23 @@ let dss_vcpus xc doms =
                     blocked"
                  ~ty:Rrd.Derive ~default:false ~min:0.0 ()
              )
+          :: ( Rrd.VM uuid
+             , Ds.ds_make ~name:"runstate_runnable" ~units:"(fraction)"
```

Comment:
I'm not sure we should call this `runstate_runnable`, as that suggests it is a unique runstate when it is actually the sum of other runstates.

I'd suggest perhaps `some_runnable` or something similar, but perhaps others have ideas?

Reply:
How about `some_waiting_for_pcpu`?

Reply:
Anything with "some" is confusing for me. If this is a fraction, "runnable/available/free_pcpu" or "pcpu_availablility" with a fractional value would make sense to me.

Reply:
Now that it is clear we are talking about vcpu, I would suggest "vcpu_availability"; traditionally this would be expressed as "vcpu_load", which would be the inverted value: vcpu_load = 1 - vcpu_availability. It is somewhat confusing to have this in addition to load. Because what is measured here?

Reply:
No, this definitely isn't load/availability, this is about the time that at least one vCPU was runnable, i.e. there was one or more vCPUs waiting to execute but due to contention (i.e. other vcpus are active on all the pcpus) it hasn't been scheduled.

Other hypervisors use the term 'ready' rather than runnable for example.

In an ideal world customers will see a 0 value here, but monitoring how high it gets on an overcommitted system can give you a good idea of how contended your system is and whether you might want to consider reducing that overcommittment

Reply:
Having one vcpu additionally available/runnable/ready would lead to the same number as having three vcpus additionally available - seems like a coarse measure. In any case, the number represents available but unused capacity in my current understanding.